.github/ISSUE_TEMPLATE.md

```diff
@@ -1,15 +1,73 @@
-(If you will report an issue about the result of parsing,
-fill the following items. If you get the input file somewhere,
-write the URL or something.)
+(
+Thank you for contacting us.
+
+*If you are reporting an issue with the parsing output, please fill
+the following template.  As your custom CTags configuration can
+affect results, please always use `--options=NONE` as the first
+option when running `ctags`.*
```

Comment:
I'd remove the italics (`*` around), I had them in my comment only to highlight the part that I suggested as replacement, sorry.

Reply:
Oh, I see.